conf/messages.en

```diff
@@ -257,8 +257,10 @@ hasCommodityCode.caption = About the goods
 hasCommodityCode.checkYourAnswersLabel = Do you have a commodity code for the goods?
 hasCommodityCode.error.required = Select yes if you know the commodity code
 hasCommodityCode.details.label = Help finding your commodity code
-hasCommodityCode.paragraph.1 = A commodity code is an internationally recognised reference number. It will help us identify the specific product you are importing. <br><br>You can {0} if you don’t know it.
-hasCommodityCode.paragraph.1.link = look up the commodity code for your goods (opens in new tab)
+hasCommodityCode.paragraph.1 = A commodity code is an internationally recognised reference number. It will help us identify the specific product you are importing.
+hasCommodityCode.linkText = look up the commodity code for your goods (opens in new tab)
+hasCommodityCode.youCan = You can
```

Comment:
small minor thing but the apostrophe in `don't` is also different which does not display when properly on the page.  Use this  `don’t`

<img width="1361" alt="Screenshot 2023-11-20 at 16 27 13" src="https://github.com/hmrc/advance-valuation-rulings-frontend/assets/47248104/aaaa93a5-de9d-4ac5-b64f-a6f3963791bf">